tests/test_client/views.py

```diff
@@ -63,6 +65,25 @@ def post_view(request):
     else:
         t = Template('Viewing GET page.', name='Empty GET Template')
         c = Context()
+    return HttpResponse(t.render(c))
+
+
+def json_view(request):
+    """
+    A view which expects a request with the header 'application/json' and
+    parseable json data to go with it.
+
+    We expect a value named 'value' in the data.
```

Comment:
Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).